standalone/src/models/shipping-method/shipping-rate/shipping-rate-price-tier/cart-value/fields-config.ts

```diff
@@ -0,0 +1,29 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { MoneyRest } from '@/models/commons';
+import type {
+  TCartValuePriceTierGraphql,
+  TCartValuePriceTierRest,
+} from './types';
+
+// https://docs.commercetools.com/api/projects/shippingMethods#cartvaluetier
+
+const commonFieldsConfig = {
+  type: 'CartValue',
+  price: fake(() => MoneyRest.random()),
+  minimumCentAmount: fake((f) => f.number.int({ min: 1000, max: 5000 })),
+  isMatching: fake((f) => f.datatype.boolean()),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TCartValuePriceTierRest> = {
+  fields: {
+    ...commonFieldsConfig,
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TCartValuePriceTierGraphql> =
+  {
+    fields: {
+      ...commonFieldsConfig,
+      __typename: 'CartValuePriceTier',
```

Comment:
Based on the TS type we're using, it seems we should be using a different value here: `ShippingRateCartValuePriceTier`